spec/requests/as_server_app/comment_thread_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'yt/models/comment_thread'
+
+describe Yt::CommentThread, :server_app do
+  subject(:comment_thread) { Yt::CommentThread.new attrs }
+
+  context 'given an existing comment thread ID about a channel' do
+    let(:attrs) { {id: 'z13vsnnbwtv4sbnug232erczcmi3wzaug'} }
+
+    it { expect(comment_thread.video_id).to be_nil }
```

Comment:
Oh, I see the example below has a video ID. 

So I think I understand what is going on here.

YouTube allows people to comment **on** a video and (separately) **on** a channel.

So probably video_id is present in one case (and channel_id missing) and vice versa.

For instance, these are "Channel comments": https://www.youtube.com/user/Fullscreen/discussion